### PR TITLE
Add Tailwind plugin to ESLint

### PR DESCRIPTION
**Pull request by torztomasz**
This pull request adds the Tailwind plugin to ESLint, which will enable linting for Tailwind CSS classes in the codebase.

### CODE REVIEW

File: packages/frontend/src/components/ExpandableContainer.tsx [hunk 1]
```diff
@@ -11,28 +11,39 @@ interface ExpandableContainerProps {
 
 export function ExpandableContainer(props: ExpandableContainerProps) {
   return (
-    <div className="ExpandableContainer">
```
Comment (antooni) [inline packages/frontend/src/components/ExpandableContainer.tsx:14]: I thought we got rid of those classes in previous PR?

**Comment by antooni** [comment]
So the only thing you have done was to add this plugin and later run it, and it resulted in this diff right?

File: packages/frontend/src/components/ExpandableContainer.tsx (hunk 1 above)
Comment (torztomasz) [reply packages/frontend/src/components/ExpandableContainer.tsx:14]: Nope, these are the classes responsible for being queryable in scripts. Previously we deleted Sass classes

**Comment by torztomasz** [comment]
> So the only thing you have done was to add this plugin and later run it, and it resulted in this diff right?

Yup, also had to manually fix some stuff

**Review by antooni — APPROVED** [review]

**Pull request merged**